### PR TITLE
translation for injector and backend

### PR DESCRIPTION
**Pull request by lilai23**
**What type of PR is this?**

Doc

**What this PR does / why we need it?**

This initiative aims to enhance readability, accessibility, and collaboration for both current and potential contributors, ensuring a unified language throughout the codebase.

**Which issue(s) this PR fixes？**

Fixes #1466

**Does this PR introduce a user-facing change?**

No

## Checklist
- [x] Make sure there is a [GitHub_issue](https://github.com/huaweicloud/Sermant/issues) related with this PR before you start working on it.
- [x] Make sure you have squashed your change to one single commit.
- [x] GitHub Actions works fine in this PR.

### CODE REVIEW

File: sermant-backend/src/main/java/com/huaweicloud/sermant/backend/common/conf/CommonConst.java [hunk 1]
```diff
@@ -17,90 +17,89 @@
...
 
     /**
-     * redis 全匹配字符*
+     * redis fully matched character *
      */
     public static final String FULL_MATCH_KEY = ".*";
 
     /**
-     * 拼接redis key 字符
+     * redis key concatenation character -
```
Comment (hanbingleixue) [inline sermant-backend/src/main/java/com/huaweicloud/sermant/backend/common/conf/CommonConst.java]: Not -, it should be _

File: sermant-injector/deployment/release/injector/values.yaml [hunk 2]
```diff
@@ -1,46 +1,51 @@
 namespace:
-  # injector部署的namespace
+  # namespace of injector
   name: default
 
 injector:
-  # 当宿主应用已存在JAVA_TOOL_OPTIONS变量时，injector的注入行为，before为在原变量前注入（默认），after为在原变量后注入，ignore为不注入
+  # When the host application already has JAVA TOOL OPTIONS variables, injector injection behavior: before indicates
```
Comment (hanbingleixue) [inline sermant-injector/deployment/release/injector/values.yaml]: JAVA_TOOL_OPTIONS

File: sermant-injector/src/main/java/com/huaweicloud/sermant/injection/certificate/CertificateEnvironmentPostProcessor.java [hunk 3]
```diff
@@ -58,12 +58,12 @@ public class CertificateEnvironmentPostProcessor implements EnvironmentPostProce
 
     @Override
     public void postProcessEnvironment(ConfigurableEnvironment environment, SpringApplication application) {
-        // k8s环境中，这个证书是base64编码后的，所以需要解码
+        // In the k8 s environment, this certificate is base64 encoded, so it needs to be decoded
```
Comment (hanbingleixue) [inline sermant-injector/src/main/java/com/huaweicloud/sermant/injection/certificate/CertificateEnvironmentPostProcessor.java]: k8s

File: sermant-injector/src/main/java/com/huaweicloud/sermant/injection/certificate/CertificateEnvironmentPostProcessor.java [hunk 4]
```diff
@@ -58,12 +58,12 @@ public class CertificateEnvironmentPostProcessor implements EnvironmentPostProce
...
         String certificatePath = getNotEmptyText(environment, CERTIFICATE_PATH_KEY, DEFAULT_CERTIFICATE_PATH);
         Optional<String> certificate = read(certificatePath);
         certificate.ifPresent(data -> write(data, certificatePath));
 
-        // k8s环境中，这个私钥是base64编码后的，所以需要解码
+        // In the k8 s environment, this private key is base64 encoded, so it needs to be decoded
```
Comment (hanbingleixue) [inline sermant-injector/src/main/java/com/huaweicloud/sermant/injection/certificate/CertificateEnvironmentPostProcessor.java]: k8s

File: sermant-injector/src/main/java/com/huaweicloud/sermant/injection/controller/SermantInjectorController.java [hunk 5]
```diff
@@ -240,64 +240,62 @@ private Response modifyRequest(ObjectNode body) {
...
-        // 缓存每个容器的环境变量
+        // Cache environment variables for each container
         Map<Integer, Map<String, String>> containerEnv = new HashMap<>();
 
-        // 缓存容器的env
+        // Cache env of container
         setEnv(containersNode, containerEnv);
 
-        // 根据labels计算需要新增env
+        // Add env based on labels
```
Comment (hanbingleixue) [inline sermant-injector/src/main/java/com/huaweicloud/sermant/injection/controller/SermantInjectorController.java]: Calculate the env to be added based on the labels.

File: sermant-injector/src/main/java/com/huaweicloud/sermant/injection/controller/SermantInjectorController.java [hunk 6]
```diff
@@ -240,64 +240,62 @@ private Response modifyRequest(ObjectNode body) {
...
 
     /**
-     *    根据labels/annotations计算需要增加的环境变量,如下以labels为例：
-     *     labels:
-     *       env.sermant.io/[key1]：[value1]
-     *       env.sermant.io/[key2]：[value2]
-     *    则在环境变量map中添加 key1:value1 和 key2:value2，共两个环境变量
-     *    本函数本真不强制是否使用labels还是annotations，只需要srcMap代表label/annotations下的map即可。
+     * Calculate environment variables that need to be added based on labels/annotations, taking labels as an example:
+     * labels: env.sermant.io/[key1]：[value1] env.sermant.io/[key2]：[value2] key1:value1 and key2:value2 are added to
```
Comment (hanbingleixue) [inline sermant-injector/src/main/java/com/huaweicloud/sermant/injection/controller/SermantInjectorController.java]: Advise to adjust the labels format

File: sermant-injector/src/main/java/com/huaweicloud/sermant/injection/controller/SermantInjectorController.java [hunk 7]
```diff
@@ -409,18 +408,19 @@ private void injectReadinessProbe(ArrayNode arrayNode, Map<String, String> env,
         boolean enableSpring = getNotEmptyValue(env, ENABLE_SPRING_KEY, true, Boolean::parseBoolean);
         boolean enableHealthCheck = getNotEmptyValue(env, ENABLE_HEALTH_CHECK_KEY, false, Boolean::parseBoolean);
 
-        // 原来已经有readinessProbe节点/periodSeconds<=0/未开启健康检查/未开启spring优雅上下线，则不注入
+        // If there is already a readiness Probe node /period Seconds<=0/ If health check is not enabled/spring
```
Comment (hanbingleixue) [inline sermant-injector/src/main/java/com/huaweicloud/sermant/injection/controller/SermantInjectorController.java]: If there is already a readinessProbe node /periodSeconds<=0

File: sermant-backend/src/main/java/com/huaweicloud/sermant/backend/common/conf/CommonConst.java [hunk 8]
```diff
@@ -17,90 +17,89 @@
...
 
     /**
-     * redis 事件key
+     * redis event key
      */
     public static final String REDIS_EVENT_KEY = "sermant_events_hash";
 
     /**
-     * redis 事件field集合的key
+     * redis event field keyset
```
Comment (daizhenyu) [inline sermant-backend/src/main/java/com/huaweicloud/sermant/backend/common/conf/CommonConst.java]: key of the redis event field set

File: sermant-backend/src/main/java/com/huaweicloud/sermant/backend/dao/EventDao.java [hunk 9]
```diff
@@ -25,59 +25,58 @@
...
-     * @return 查询结果
+     * @param sessionId session id
+     * @param page page number
+     * @return query result
      */
     List<QueryResultEventInfoEntity> queryEventPage(String sessionId, int page);
 
     /**
-     * 获取查询结果数据量
+     * Query the number of data types
```
Comment (daizhenyu) [inline sermant-backend/src/main/java/com/huaweicloud/sermant/backend/dao/EventDao.java]: get the cache size in the query result.

File: sermant-backend/src/main/java/com/huaweicloud/sermant/backend/dao/EventDao.java [hunk 10]
```diff
@@ -25,59 +25,58 @@
...
      *
-     * @param eventsRequestEntity 查询参数
-     * @return 查询结果数据量
+     * @param eventsRequestEntity Event request entity
+     * @return query result
```
Comment (daizhenyu) [inline sermant-backend/src/main/java/com/huaweicloud/sermant/backend/dao/EventDao.java]: the cache size in query result

File: sermant-backend/src/main/java/com/huaweicloud/sermant/backend/dao/redis/EventDaoForRedis.java [hunk 11]
```diff
@@ -79,33 +79,33 @@ public boolean addInstanceMeta(InstanceMeta instanceMeta) {
...
     }
 
     /**
-     * 获取查询数据类型数量
+     * Query the number of data types
      *
-     * @param eventsRequestEntity 事件请求体
-     * @return 查询结果数量
+     * @param eventsRequestEntity Event request entity
+     * @return query result
```
Comment (daizhenyu) [inline sermant-backend/src/main/java/com/huaweicloud/sermant/backend/dao/redis/EventDaoForRedis.java]: Confirm the translation. Keep it consistent with the same method above.

File: sermant-backend/src/main/java/com/huaweicloud/sermant/backend/entity/visibility/ServerInfo.java [hunk 12]
```diff
@@ -22,84 +22,84 @@
...
 public class ServerInfo extends BaseInfo {
     /**
-     * 区域
+     * Zone
      */
     private String zone;
 
     /**
-     * 命名空间
+     * Project
```
Comment (daizhenyu) [inline sermant-backend/src/main/java/com/huaweicloud/sermant/backend/entity/visibility/ServerInfo.java]: Namespace

File: sermant-backend/src/main/java/com/huaweicloud/sermant/backend/webhook/welink/WelinkHookClient.java [hunk 13]
```diff
@@ -24,38 +24,37 @@
 import java.util.List;
 
 /**
- * welink webhook 客户端
+ * Celink webhook client
```
Comment (daizhenyu) [inline sermant-backend/src/main/java/com/huaweicloud/sermant/backend/webhook/welink/WelinkHookClient.java]: welink

File: sermant-injector/src/main/resources/application.yml [hunk 14]
```diff
@@ -6,36 +6,37 @@ server:
...
     type: SERVICE_COMB
-    # agent注册中心地址
+    # Registration center address
     address: http://localhost:30100
-  # sermant-agent 配置环境变量的configMap的名字
+  # The name of the configMap that configures the environment variable in sermant-agent
   configMap: sermant-agent-env
   inject:
-    # 当宿主应用已存在JAVA_TOOL_OPTIONS变量时，sermant的注入行为，before为在原变量前注入（默认），after为在原变量后注入，ignore为不注入
+    # Always(always pull), IfNotPresent(default value, local image is used if there is a local image, no pull is used)
```
Comment (daizhenyu) [inline sermant-injector/src/main/resources/application.yml]: Confirm the translation.

File: sermant-backend/src/main/java/com/huaweicloud/sermant/backend/common/conf/CommonConst.java (hunk 1 above)
Comment (lilai23) [reply sermant-backend/src/main/java/com/huaweicloud/sermant/backend/common/conf/CommonConst.java]: done

File: sermant-injector/src/main/java/com/huaweicloud/sermant/injection/certificate/CertificateEnvironmentPostProcessor.java (hunk 3 above)
Comment (lilai23) [reply sermant-injector/src/main/java/com/huaweicloud/sermant/injection/certificate/CertificateEnvironmentPostProcessor.java]: done

File: sermant-injector/deployment/release/injector/values.yaml (hunk 2 above)
Comment (lilai23) [reply sermant-injector/deployment/release/injector/values.yaml]: done

File: sermant-injector/src/main/java/com/huaweicloud/sermant/injection/certificate/CertificateEnvironmentPostProcessor.java (hunk 4 above)
Comment (lilai23) [reply sermant-injector/src/main/java/com/huaweicloud/sermant/injection/certificate/CertificateEnvironmentPostProcessor.java]: done

File: sermant-injector/src/main/java/com/huaweicloud/sermant/injection/controller/SermantInjectorController.java (hunk 5 above)
Comment (lilai23) [reply sermant-injector/src/main/java/com/huaweicloud/sermant/injection/controller/SermantInjectorController.java]: done

File: sermant-injector/src/main/java/com/huaweicloud/sermant/injection/controller/SermantInjectorController.java (hunk 6 above)
Comment (lilai23) [reply sermant-injector/src/main/java/com/huaweicloud/sermant/injection/controller/SermantInjectorController.java]: done

File: sermant-injector/src/main/java/com/huaweicloud/sermant/injection/controller/SermantInjectorController.java (hunk 7 above)
Comment (lilai23) [reply sermant-injector/src/main/java/com/huaweicloud/sermant/injection/controller/SermantInjectorController.java]: done

File: sermant-backend/src/main/java/com/huaweicloud/sermant/backend/common/conf/CommonConst.java (hunk 8 above)
Comment (lilai23) [reply sermant-backend/src/main/java/com/huaweicloud/sermant/backend/common/conf/CommonConst.java]: done

File: sermant-backend/src/main/java/com/huaweicloud/sermant/backend/dao/EventDao.java (hunk 9 above)
Comment (lilai23) [reply sermant-backend/src/main/java/com/huaweicloud/sermant/backend/dao/EventDao.java]: done

File: sermant-backend/src/main/java/com/huaweicloud/sermant/backend/dao/EventDao.java (hunk 10 above)
Comment (lilai23) [reply sermant-backend/src/main/java/com/huaweicloud/sermant/backend/dao/EventDao.java]: done

File: sermant-backend/src/main/java/com/huaweicloud/sermant/backend/entity/visibility/ServerInfo.java (hunk 12 above)
Comment (lilai23) [reply sermant-backend/src/main/java/com/huaweicloud/sermant/backend/entity/visibility/ServerInfo.java]: done

File: sermant-backend/src/main/java/com/huaweicloud/sermant/backend/webhook/welink/WelinkHookClient.java (hunk 13 above)
Comment (lilai23) [reply sermant-backend/src/main/java/com/huaweicloud/sermant/backend/webhook/welink/WelinkHookClient.java]: done

File: sermant-injector/src/main/resources/application.yml (hunk 14 above)
Comment (lilai23) [reply sermant-injector/src/main/resources/application.yml]: done

File: sermant-backend/src/main/java/com/huaweicloud/sermant/backend/dao/redis/EventDaoForRedis.java (hunk 11 above)
Comment (lilai23) [reply sermant-backend/src/main/java/com/huaweicloud/sermant/backend/dao/redis/EventDaoForRedis.java]: done

**Review by hanbingleixue — APPROVED** [review]

**Review by daizhenyu — APPROVED** [review]

**Pull request merged**